### PR TITLE
Feature/remove sdx home in makefile

### LINKED ISSUE #58: Makefile should not be referencing SDX_HOME env var
The `SDX_HOME` env var is not required by the service and causes travis build to fail as it's not set.

### PR DESCRIPTION
**Pull request by gemmai95**
Fixes #58

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,12 @@
-dev: check-env
+dev:
 	cd .. && pip3 uninstall -y sdx-common && pip3 install -I ./sdx-common
```
Comment (necrophonic) [inline Makefile]: Running a standard ``make; make test`` doesn't work as it assumes you already have ``sdx-common`` installed (which you won't on a fresh checkout) and make fails.

``build`` should be the default ``make`` step but that will also fail if you remove the ``git clone`` step.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
   - Addings sdx-common functionality
   - updating timezoning
   - Update and pin version of sdx-common to 0.7.0
+  - Remove SDX_HOME env Variable from makefile
```
Comment (necrophonic) [inline CHANGELOG.md]: More's changed in this PR than just removing the ``SDX_HOME`` var

**Comment by warrenbailey** [comment]
It's also worth adding Fixes #58 in the PR description as that automatically closes the github issue when this merged.

**Comment by gemmai95** [comment]
[(2) Remove erroneous use of SDX_HOME in makefiles](https://trello.com/c/yaTznQmn/190-2-remove-erroneous-use-of-sdxhome-in-makefiles)

File: Makefile (hunk 1 above)
Comment (gemmai95) [inline Makefile]: Would the git clone be better in the makefile rather than the travis file?

**Comment by necrophonic** [comment]
Superseded by cherry picked commits into #64 to clean git history

**Pull request closed without merging**